nimterop/build.nim

```diff
@@ -4,6 +4,15 @@ import os except findExe, sleep
 
 import regex
 
+type
+  MakeType* = enum
+    mtConfMake, mtCMake
```

Comment:
How about calling this `BuildType` and the options `autoconf` and `cmake`?

Reply:
👍

Reply:
Can't use `cmake` because it conflicts with proc names. Named it `btAutoconf` and `btCmake`